rosdep/python.yaml

```diff
@@ -2859,6 +2859,17 @@ python-pyaudio:
   fedora: [pyaudio]
   gentoo: [dev-python/pyaudio]
   ubuntu: [python-pyaudio]
+python-pycurl:
+  debian:
+    buster: [python-pycurl]
+    jessie: [python-pycurl]
+    sid: [python-pycurl]
+    stretch: [python-pycurl]
+  ubuntu:
```

Comment:
Although trusty doesn't have this packaged we could phrase this as:

```
  ubuntu:
    '*': [python-pycurl]
    trusty: null
```

But I don't strongly believe this change is required.

Reply:
Trusty is EOL within a month. I'd suggest saving the extra line.